charts/gardener/controlplane/values.yaml

```diff
@@ -14,8 +14,8 @@ global:
         cpu: 100m
         memory: 100Mi
       limits:
-        cpu: 300m
-        memory: 256Mi
+        cpu: "8"
+        memory: 25G
```

Comment:
Why are the limits so high? Is this required for hvpa?
Is my assumption right, that the apiserver Pods will then have 8/25G limits initially right away?
In that case, the range `[requests,limits]` seems a bit too large IMO.

Reply:
The last release of HVPA controller doesn't scale `limits`, that's why it is kept very high.  In the next release of HVPA, there will be some `spec` changes along with scaling of `limits`. These values will be adopted accordingly when Gardener starts using the next release.